xclim/sdba/utils.py

```diff
@@ -417,35 +422,39 @@ def interp_on_quantiles(
       If it does, interpolation is done in 2D on "quantiles" and on the group dimension.
     group : Union[str, Grouper]
       The dimension and grouping information. (ex: "time" or "time.month").
-      Defaults to the "group" attribute of xq, or "time" if there is none.
+      Defaults to "time".
     method : {'nearest', 'linear', 'cubic'}
-      The interpolation method.
+      The interpolation method. Using cubic interpolation will NOT respect the constant
+      extrapolation as added by :py:func:`~xclim.sdba.utils.extrapolate_qm`.
```

Comment:
You mean `interp_on_quantiles` will in fact extrapolate ? And this extrapolation is going to be cubic instead of constant. If the extrapolation used nan, then it's ok ? The cubic extrapolation will still be nan, right ?

Reply:
Yes, it will extrapolate. 'cubic' uses 3 points, It means the fitted spline for out-of-bounds data is not flat as it includes the second-to-last real point in addition to the last and its copy. I didn't try with `nan`, will do.

Reply:
Woups. When `extrapolate='nan', method='cubic'`, output is NaN everywhere.

Well. I'll fix that too.